lib/mj-page.js

```diff
@@ -401,7 +400,6 @@ function ConfigureMathJax() {
               }
               //
               //  Mark math for screen readers
-              //    (screen readers don't know about role="math" yet, so use "textbox" instead)
```

Comment:
The "Mark math for screen readers" should be removed as well, since that is no longer being done.  So this whole comment can go.